docs/internals/howto-release-django.txt

```diff
@@ -222,28 +233,28 @@ OK, this is the fun part, where we actually push out a release!
     package and compare them to the checksums listed below.
 
 
-    Release package:
-    ================
+    Release packages:
+    =================
 
     Django <<VERSION>>: https://www.djangoproject.com/m/releases/<<URL>>
 
 
-    MD5 checksum:
-    =============
+    MD5 checksums:
+    ==============
 
     MD5(<<RELEASE TAR.GZ FILENAME>>)= <<MD5SUM>>
```

Comment:
do we need to use this funky format? Can we just copy/paste the output of md5sum, etc?
```
bdda6ad32fecff67423a720595906503  Django-1.8-py2.py3-none-any.whl
68077000c8063a7478ece2f55a53d0bf  Django-1.8.tar.gz
```